src/main/java/it/gov/pagopa/afm/calculator/repository/CosmosRepository.java

```diff
@@ -88,7 +88,11 @@ private static boolean digitalStampFilter(
                 bundle.getDigitalStampRestriction() != null
                         ? bundle.getDigitalStampRestriction()
                         : Boolean.FALSE;
-        if (onlyMarcaBolloDigitale == transferListSize) {
+        if(transferListSize == 0 && onlyMarcaBolloDigitale == 0){
```

Comment:
I would add a junit test to cover this specific filtering condition, what do you think?